docs/ConfigsAndApplications.md

```diff
@@ -12,6 +12,10 @@ with the syntax and meaning of component trees, it is a good idea to browse the
 Infusion [documentation site](http://docs.fluidproject.org/infusion/development/). Kettle components are currently derived from
 the base grade `fluid.component`, so you can ignore for these purposes the parts of the Infusion documentation relating to model and view components.
 
+Note that Kettle configs need not code for Kettle applications. Their structure is
```

Comment:
What does it mean by "Kettle configs need not code for Kettle applications"? Does it mean "Kettle configs are not needed for coding Kettle applications"? Excuse my ignorance in English if this is a silly question. ;)